docs/source/about/guides/linux/headless_ssh.rst

```diff
@@ -250,125 +249,102 @@ As an alternative to a dummy dongle, you can use this config to create a virtual
+#. Create alternate *udev* rules.
 
-#. Create a ``sunshine-setup.sh`` script to update permissions on ``/dev/uinput``. Since we aren't logged into the host,
-   the udev rule doesn't apply.
-#. Update user sudo configuration ``/etc/sudoers.d/<user>`` to allow the ``sunshine-setup.sh``
-   script to be executed with ``sudo``.
+   .. note::
+      These alternative rules are necessary because the original rules only seem to trigger on physical login and not ssh.
+      These rules open access to uinput to the input group.
 
-.. note::
-   After I setup the :ref:`udev rule <about/setup:install>` to get access to ``/dev/uinput``,
-   I noticed when I sshed into the host without physical login, the ACL permissions on ``/dev/uinput`` were not changed.
-   So I asked `reddit
-   <https://www.reddit.com/r/linux_gaming/comments/14htuzv/does_sshing_into_host_trigger_udev_rule_on_the/>`__.
-   I discovered that SSH sessions are not the same as a physical login.
-   I suppose it's not possible for SSH to trigger a udev rule or create a physical login session.
-
-**Setup Script**
-
-This script will take care of any preconditions prior to starting up sunshine.
-
-Run the following to create a script named something like ``sunshine-setup.sh``:
-   .. code-block:: bash
-
-      echo "chown $(id -un):$(id -gn) /dev/uinput" > sunshine-setup.sh &&\
-        chmod +x sunshine-setup.sh
-
-(**Optional**) To Ensure ethernet is being used for streaming,
-you can block WiFi with ``rfkill``.
-
-Run this command to append the rfkill block command to the script:
    .. code-block:: bash
 
-      echo "rfkill block $(rfkill list | grep "Wireless LAN" \
-        | sed 's/^\([[:digit:]]\).*/\1/')" >> sunshine-setup.sh
-
-**Sudo Configuration**
-
-We will manually change the permissions of ``/dev/uinput`` using ``chown``.
-You need to use ``sudo`` to make this change, so add/update the entry in ``/etc/sudoers.d/${USER}``
+      echo 'KERNEL=="uinput", SUBSYSTEM=="misc", MODE="0660", GROUP="input", OPTIONS+="static_node=uinput"' | \
+      sudo tee /etc/udev/rules.d/85-sunshine.rules
 
-.. danger::
-   Do so at your own risk! It is more secure to give sudo and no password prompt to a single script,
-   than a generic executable like chown.
-
-.. warning::
-   Be very careful of messing this config up. If you make a typo, *YOU LOSE THE ABILITY TO USE SUDO*.
-   Fortunately, your system is not borked, you will need to login as root to fix the config.
-   You may want to setup a backup user / SSH into the host as root to fix the config if this happens.
-   Otherwise you will need to plug your machine back into a monitor and login as root to fix this.
-   To enable root login over SSH edit your SSHD config, and add ``PermitRootLogin yes``, and restart the SSH server.
-
-#. First make a backup of your ``/etc/sudoers.d/${USER}`` file.
+#. Add the user running sunshine to the `input` group.
```

Comment:
This used to be part of our instructions... not sure when or why it got removed.